src/Common/Common.Api/Workspaces/WorkspacesClient.cs

```diff
@@ -16,49 +16,49 @@ public WorkspacesClient(IPowerBIClient client) : base(client)
         {
         }
 
-        public object AddWorkspaceUser(Guid workspaceId, WorkspaceUser userAccessRight)
+        public void AddWorkspaceUser(Guid workspaceId, WorkspaceUser userAccessRight)
         {
-            return this.Client.Groups.AddGroupUser(workspaceId.ToString(), userAccessRight);
+            this.Client.Groups.AddGroupUser(workspaceId, userAccessRight);
         }
 
-        public object AddWorkspaceUserAsAdmin(Guid workspaceId, WorkspaceUser userAccessRight)
+        public void AddWorkspaceUserAsAdmin(Guid workspaceId, WorkspaceUser userAccessRight)
         {
-            return this.Client.Groups.AddUserAsAdmin(workspaceId.ToString(), userAccessRight);
+            this.Client.Groups.AddUserAsAdmin(workspaceId, userAccessRight);
         }
 
         public IEnumerable<Workspace> GetWorkspaces(string filter = null, int? top = null, int? skip = null)
         {
             return this.Client.Groups.GetGroups(filter, top, skip).Value.Select(x => (Workspace)x);
         }
 
-        public IEnumerable<Workspace> GetWorkspacesAsAdmin(string expand = null, string filter = null, int? top = null, int? skip = null)
+        public IEnumerable<Workspace> GetWorkspacesAsAdmin(int top, string expand = null, string filter = null, int? skip = null)
         {
-            return this.Client.Groups.GetGroupsAsAdmin(expand, filter, top, skip).Value.Select(x => (Workspace)x);
+            return this.Client.Groups.GetGroupsAsAdmin(top, expand, filter, skip).Value.Select(x => (Workspace)x);
```

Comment:
These parameters aren't named like other wrapped calls to the SDK so I'm not sure if the reordering is intentional or not.